src/screens/Survey/index.js

```diff
@@ -10,7 +10,7 @@ class SurveyMainScreen extends React.Component {
   render() {
     return (
       <View style={styles.container}>
-        <Button title="Go to District Match" onPress={this.goToDistrictMatch}/>
+        <Button title="Go to District Match" onPress={this.goToDistrictMatch} />
```

Comment:
I think if you just take this space out, your merge conflict will go away, because you didn't actually change the content of this file, which is probably easier than actually solving it using a merge tool.  You could also look up git merge strategy (I only know how to use it in the rebase flow, which I'm not sure if you are using, but you can git rebase -i develop, then when it complains about the merge conflict, git checkout scr/screens/Survey/index.js -ours to use the updated one on develop)

Reply:
Sorry this is my bad from my editor's auto formatting. Mia and I went through an interactive rebase together to get rid of the old commits on her branch. That's a reason not to squash merge by the way. If all of those commits were in the develop branch, git would have been able to separate out just the new content

Reply:
Yeah, definitely a formatting change.  I think this whole screen was replaced though, so that button does not exist anymore.  I am generally a fan of interactive rebase flow, in this case, I thought just removing the change might be the easiest to just remove the change so that git would just know the new version was correct.  It actually looks like the index file might have been deleted here altogether, so you should make sure you are not adding back in the old file that is not used anymore